kuma/users/utils.py

```diff
@@ -46,14 +46,21 @@ def retrieve_stripe_subscription_info(user):
     )
     if stripe_subscription:
         source = stripe_customer.default_source
```

Comment:
What other sources are there? I.e. other than `default_source`

Reply:
`stripe_customer` also has a list of sources one could iterate. But given that we control how sources are created, there are no other sources than Cards to be expected (BankAccounts would be another theoretical option)